.github/actions/test-blueprint/action.yml

```
@@ -35,6 +35,18 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Free Disk Space (Ubuntu)
+      if: startsWith(inputs.os, 'ubuntu-')
+      uses: jlumbroso/free-disk-space@main
+      with:
+        tool-cache: false
+        android: true
+        dotnet: true
+        haskell: true
+        large-packages: true
+        docker-images: true
+        swap-storage: false
```

```suggestion
        swap-storage: false
```
The removed ones are the default anyway, c.f. https://github.com/jlumbroso/free-disk-space/blob/main/action.yml